app/assets/javascripts/characters.coffee

```diff
@@ -0,0 +1,38 @@
+$ ->
+  $(document).on 'click', '#character_career_id_chosen', (evt) ->
+    $.ajax 'skills_select',
+      type: 'GET'
+      dataType: 'script'
+      data: {
+        career: $("#character_career_id_chosen > a > span").text()
+      }
+      error: (jqXHR, textStatus, errorThrown) ->
+        console.log("AJAX Error: #{textStatus}")
```

Comment:
`console.log` ? there is no point on doing this....you either remove the whole `error` callback, and the same goes for the `success`